mne_bids/path.py

```diff
@@ -142,6 +145,10 @@ def _find_matched_empty_room(bids_path):
     return best_er_bids_path
 
 
+@pytest.mark.xfail(condition="os.name == 'nt'",
+                   reason='doctest fails on Windows due to os.sep difference',
```

Comment:
Wait, shouldn't we assure that the `str()` and `repr()` are the same on all platforms?

Reply:
but would the paths then still work on Windows?

Reply:
maybe like this?

```diff
    def __str__(self):
        """Return the string representation of the path."""
-        return str(self.fpath)
+       return str(self.fpath.as_posix()) 
```

Reply:
I just pushed it. We'll see if it works, I think that's a good idea and simplifies the whole thing. And BIDS wants only forward slashes anyhow :-) Fingers crossed that the test suite works

Reply:
> but would the paths then still work on Windows?

My idea is that we wouldn't touch .fpath, just change the way we str() and repr() things :)

Reply:
that solved the first problem - thanks!